pkg/controllers/externalsecret/externalsecret_controller_test.go

```diff
@@ -130,6 +112,24 @@ var _ = Describe("Kind=secret existence logic", func() {
 	}
 })
 var _ = Describe("ExternalSecret controller", func() {
+	type testCase struct {
+		secretStore    *esv1beta1.SecretStore
```

Comment:
Could you please make this a `esv1beta1.GenericStore` (that's an interface) - that way we should be able to re-use the test setup regardless if we use a `SecretStore` or `ClusterSecretStore`.

Reply:
Done :)